applications/acdc/src/acdc_agent_fsm.erl

```diff
@@ -1703,19 +1703,19 @@ notify(Url, 'get', Data) ->
 
 -spec notify(iolist(), [], 'get' | 'post', binary(), wh_proplist()) -> 'ok'.
 notify(Uri, Headers, Method, Body, Opts) ->
-    case ibrowse:send_req(wh_util:to_list(Uri)
-                          ,Headers
-                          ,Method
-                          ,Body
-                          ,[{'connect_timeout', 200} % wait up to 200ms for connection
-                            | Opts
-                           ]
-                          ,1000
-                         )
+    case kz_http:req(Method
```

Comment:
`Method` here is either `post` or `get`. I think there is no much gain to rewrite and merge the `notify/3` and `notify/4` functions.